Lib/platform.py

```diff
@@ -334,6 +334,30 @@ def _syscmd_ver(system='', release='', version='',
     (6, None): "post2012ServerR2",
 }
 
+def win32_is_iot(isIot=False):
+    if win32_editionId() in ('IoTUAP', 'NanoServer', 'WindowsCoreHeadless', 'IoTEdgeOS'):
```

Comment:
`return win32_edition() in ('IoTUAP' …`

Reply:
other point, `isIot` is not used, is it useful?

Reply:
fixed both